DOCUMENTATION.md

```diff
@@ -94,6 +94,16 @@ sms_sender_id='8e222534-7f05-4972-86e3-17c5d9f894e2' # optional UUID string
 
 You can leave out this argument if your service only has one text message sender, or if you want to use the default sender.
 
+#### `statusCallbackUrl`
```

Comment:
make all these snake_case

`status_callback_url` and `status_callback_bearer_token`

Reply:
done